docs/AboutUs.md

```diff
@@ -2,8 +2,8 @@
 
 Display | Name | Github Profile | Portfolio 
 --------|:----:|:--------------:|:---------:
-![](https://via.placeholder.com/100.png?text=Photo) | GuoAi | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Zeyu Cao | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![MuhammadPhoto](./displayPictures/MuhammadPhoto.jpg) | Muhammad Hozefa | [Github](https://github.com/MuhammadHoze) | [Portfolio](https://github.com/AY2021S1-CS2113-T14-3/tp/blob/master/docs/team/muhammadhozefa.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Chen Jia Jun | [Github](https://github.com/iamchenjiajun) | [Portfolio](docs/team/iamchenjiajun.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Antonio | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
+![](https://via.placeholder.com/100.png?text=Photo) | GuoAi | [Github](https://github.com/) | [Portfolio](./team/template.md)
+![](https://via.placeholder.com/100.png?text=Photo) | Cao Zeyu | [Github](hhttps://github.com/Cao-Zeyu) | [Portfolio](./team/Cao-Zeyu.md)
```

Comment:
I think @Cao-Zeyu  forgot to remove the extra `h' from `https://`. Could you help omit the extra `h` out. thanks

Reply:
Alright, updated 👍